Shared/Samples/Analyze network with subnetwork trace/AnalyzeNetworkWithSubnetworkTraceView.swift

```diff
@@ -0,0 +1,267 @@
+                            Text(value.name)
+                                .frame(maxWidth: .infinity, alignment: .trailing)
+                        }
+                    }
+                }
+                .disabled(model.selectedAttribute == nil && model.selectedComparison == nil)
+            } else {
+                HStack {
+                    Text("Value")
+                    Spacer()
+                    if let value = inputValue {
+                        Text(value.formatted())
+                    }
+                }
+                .contentShape(Rectangle())
+                .disabled(model.selectedAttribute == nil && model.selectedComparison == nil)
+                .onTapGesture {
+                    inputBoxIsPresented = true
+                }
+                .alert("Provide a comparison value", isPresented: $inputBoxIsPresented, actions: {
+                    TextField("10", value: $inputValue, format: .number)
+                        .keyboardType(.numberPad)
+                    Button("Done") {
+                        inputBoxIsPresented = false
+                        model.selectedValue = inputValue
+                    }
+                    Button("Cancel") {
+                        inputBoxIsPresented = false
+                        model.selectedValue = nil
+                        inputValue = nil
+                    }
+                })
+            }
+        }
+        .navigationTitle("Add Condition")
+        .navigationBarTitleDisplayMode(.inline)
+        .toolbar {
+            ToolbarItem(placement: .confirmationAction) {
+                Button("Done") {
+                    isConditionMenuPresented.toggle()
+                    model.addConditionalExpression()
+                    model.selectedValue = nil
+                    inputValue = nil
+                }
+                .disabled(
+                    model.selectedAttribute == nil ||
+                    model.selectedComparison == nil ||
+                    model.selectedValue == nil
+                )
+            }
+            ToolbarItem(placement: .cancellationAction) {
+                Button("Cancel") {
+                    isConditionMenuPresented.toggle()
+                    model.clearSelection()
+                    inputValue = nil
+                }
+            }
+        }
+        .navigationViewStyle(.stack)
+    }
+    
+    @ViewBuilder var attributesView: some View {
+        List(model.possibleAttributes, id: \.name) { attribute in
+            HStack {
+                Text(attribute.name)
+                Spacer()
+                if attribute === model.selectedAttribute {
+                    Image(systemName: "checkmark")
+                        .foregroundColor(.accentColor)
+                }
+            }
+            .contentShape(Rectangle())
+            .onTapGesture {
+                model.selectedAttribute = attribute
+            }
+        }
```

Comment:
This view could do with a title.

```suggestion
        }
        .navigationTitle("Attributes")
```